docs/source/usage.rst

```diff
@@ -136,6 +141,13 @@ below the frame window on the left.
 
 ..  figure:: ./images/use/pick_frame.png
 
+If you are interested in limited region, ROI provide option to select rectangle,
```

Comment:
Suggestion: "If you are interested in **a** limited region, **the** ROI **dropdown** provide**s**  **the** option to select **a** rectang**ular** ~~disk~~ region